fs/fs.go

```diff
@@ -428,7 +440,7 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 			ctx := log.WithLogger(context.Background(), log.G(ctx).WithField("mountpoint", mountpoint))
 			sociDesc, ok := c.imageLayerToSociDesc[desc.Digest.String()]
 			if !ok {
-				log.G(ctx).Debugf("unable to resolve layer %q, mounting entire layer using overlayfs: %s", desc.Digest, "no ztoc for the layer")
+				log.G(ctx).Debugf("unable to resolve layer %q, mounting entire layer using overlayfs: %v", desc.Digest, snapshot.ErrNoZtoc)
```

Comment:
line 420 uses `%w` for `ErrNoZtoc` and here is using `%v`.

Reply:
Unlike line 420, we're just logging the error, not returning it.